readthedocs/settings/base.py

```diff
@@ -99,6 +99,12 @@ class CommunityBaseSettings(Settings):
         "/admin/",
     )
 
+    # Permissions Policy
+    # https://github.com/adamchainz/django-permissions-policy
+    PERMISSIONS_POLICY = {
```

Comment:
Is this something we can easily test? Would be good to have at least an integration test to make sure the setting, etc. is correct.

Reply:
Are you thinking a validatehttp check? Or a pagerduty/CF check? I think those would be the best places for it.

Reply:
I think a regular Django test + validatehttp are 💯